cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/5_version_decouple.spec.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('Create datasources', () => {
+    before(() => {
+      // Clean up before creating new data sources for testing
+      cy.deleteAllDataSources();
+      cy.createDataSourceNoAuthWithTitle('ds_1');
+      cy.wait(6000);
+    });
+
+    after(() => {
+      // Clean up after all test are run
+      cy.deleteAllDataSources();
+      // remove the default data source
+      cy.setAdvancedSetting({
+        defaultDataSource: '',
+      });
+    });
+
+    describe('Check datasource contains version decouple related information', () => {
+      it('check installed plugins and data source version information is showed', () => {
+        cy.request({
+          method: 'GET',
+          url: '/api/saved_objects/_find?type=data-source',
+        }).then((response) => {
+          const savedObjects = response.body.saved_objects;
+          const dataSource = savedObjects.find(
+            (ds) => ds.type === 'data-source'
+          );
```

Comment:
This might not be necessary,  In the request you already specify type as data source `url: '/api/saved_objects/_find?type=data-source',`

Reply:
Oh yeah. Let me remove it.